### DISCUSSION TITLE
Plots of total energy vs volume

### DISCUSSION THREAD

**Discussion by diegorasero** [nguyen-group/QE-SSP#11] [General]
Hello everyone,

I have been following the examples in your book and have managed to reproduce several of the results (values and graphs).
However, in the literature I have found plots of total energy vs volume where a fit is made to an equation of state, for example, the Murnaghan equation of state.
Can this be done in QE?
How can such plots be obtained with QE?
I imagine there must be a relation to a vc-relax calculation but I have not been able to make such plots.

Best regards

**Comment by nguyen-group** [comment]
Hi,

In the QE package, ev.x can fit energy-vs-volume data to an equation of state.
In our book, we did not make the tutorial with ev.x, but it can be done in two steps:

Step 1: Prepare a text file with two columns, the first column being the unit cell volume (a.u^3 or Å^3) of each of your calculations, the second column being the corresponding total energy (Ry).
Step 2: Run ev.x in the terminal.

More detail in: https://mattermodeling.stackexchange.com/questions/3528/how-to-use-the-ev-x-tool-in-quantum-espresso

Best,
Nguyen

**Comment by diegorasero** [comment]
Thanks for the directions. I will try to follow the indicated steps.Enviado desde mi Huawei de Claro.-------- Mensaje original --------De: "Quantum Materials @ Tohoku University" ***@***.***>Fecha: lun., 16 dic. 2024, 2:23 a. m.Para: nguyen-group/QE-SSP ***@***.***>CC: diegorasero ***@***.***>, Author ***@***.***>Asunto: Re: [nguyen-group/QE-SSP] Plots of total energy vs volume (Discussion #11)
Hi,
In the QE package, ev.x can fit energy-vs-volume data to an equation of state.
In our book, we did not make the tutorial with ev.x, but it can be done in two steps:
Step 1: Prepare a text file with two columns, the first column being the unit cell volume (a.u^3 or Å^3) of each of your calculations, the second column being the corresponding total energy (Ry).
Step 2: Run ev.x in the terminal.
More detail in: https://mattermodeling.stackexchange.com/questions/3528/how-to-use-the-ev-x-tool-in-quantum-espresso
Best,
Nguyen
